.github/workflows/ngsa-pr.yaml

```diff
@@ -27,16 +27,45 @@ jobs:
         registry: ghcr.io
         username: ${{ secrets.GHCR_ID }}
         password: ${{ secrets.GHCR_PAT }}
-
+    - name: Set Secrets
+      run: |
+            echo -n ${{ secrets.NGSA_PRE_COSMOS_RO_KEY}} >| secrets/CosmosKey
```

Comment:
you just put the secret in the docker image. If you publish the image, you have a security hole.

The secrets need to be copied to a local directory and mounted with -v

Reply:
security hole fixed, run ngsa-app container with in-memory flag instead.